frontend/packages/jobs/src/prompts/generateReview/generateReview.ts

```diff
@@ -24,23 +24,20 @@ Your JSON-formatted response must contain:
     - Performance Impact
     - Project Rules Consistency
     - Security or Scalability
-  - "value": A numeric score from 0 to 10, using a deduction-based approach:
-    - Start with a perfect score of 10 (indicating no issues)
-    - Deduct points based on the severity and number of issues identified in each category
-    - Consider the cumulative impact of multiple issues
-    - A score of 0-3 indicates critical, unfixable problems
-    - A score of 4-6 indicates significant issues requiring attention
-    - A score of 7-9 indicates minor issues or concerns
-    - A score of 10 indicates perfect design with no issues
-  - "reason": An explanation justifying the score provided, including what deductions were made and why. If no issues were found in a category, explicitly state that a score of 10 was given because no issues were identified.
-
-- Based on the scores you've assigned, create an array of identified feedback in the "feedbacks" field, each including:
+  - "value": This field will be calculated by the system. Only provide the "kind" and "reason" fields.
+  - "reason": An explanation justifying the score provided, including an overview of identified issues. If no issues were found in a category, explicitly state that a score of 10 was given because no issues were identified.
+
+- Based on the findings, create an array of identified feedback in the "feedbacks" field. Each feedback item must include:
   - "kind": The feedback category, matching one of the categories used in the scores.
-  - "severity": For each feedback item, assign a severity value:
-    - Use "CRITICAL" for issues that caused major point deductions (2-4 points)
-    - Use "WARNING" for issues that caused minor point deductions (1-2 points)
-    - Use "POSITIVE" to highlight improvements, best practices, or beneficial changes in the schema design. When assigning a score of 10 in any category, at least one POSITIVE feedback must be included to justify the score.
-  - "description": A clear and precise explanation of the feedback. If the severity is POSITIVE, describe what is improved and why it is beneficial.
+  - "severity": Assign a severity value:
+    - Use "CRITICAL" for major issues.
+    - Use "WARNING" for minor issues.
+    - Use "POSITIVE" to highlight improvements in the schema design.
```

Comment:
💭 With this notation, it looked like positive would be used if there was a problem, but I think it's ok if it's tested and ok.

Reply:
The langfuse test returned feedback as intended.
```json
    {
      "kind": "Performance Impact",
      "severity": "POSITIVE",
      "description": "The introduction of sequences for ID management is a good practice for performance.",
      "suggestion": "No suggestions needed."
    },
```

If the output is not an improved point but a point that needs to be corrected, I will try to correct it!🔧

Reply:
Thanks for checking!! I understand.